tests/trezor/trezor_tests.cpp

```diff
@@ -360,6 +381,7 @@ static void setup_chain(cryptonote::core * core, gen_trezor_base & trezor_base,
     {
       trezor_base.clear();
       generated = trezor_base.generate(events);
+      trezor_base.fix_hf(events);
```

Comment:
optimization: generate blockchain file up to the minimal HF version being tested. Previously was this step performed at runtime after loading the chain file.